packages/builder/src/stores/portal/licensing.js

```diff
@@ -205,9 +204,10 @@ export const createLicensingStore = () => {
         const userCount = usage.usageQuota.users
         const userLimitReached = usersLimitReached(userCount, userLimit)
         const userLimitExceeded = usersLimitExceeded(userCount, userLimit)
-        const days = dayjs(userQuota?.startDate).diff(dayjs(), "day")
-        const userLimitDays = days > 1 ? `${days} days` : "1 day"
-        const errUserLimit = userQuota?.startDate && userLimitExceeded
+        const errUserLimit = license.account.hosting === Hosting.CLOUD &&
```

Comment:
`account` doesn't exist on the license, I think we will need to check the hosting that comes back from the `/environment` request here as an alternative. We should be able to import the `admin` store here which exports the `cloud` property like:

```ts
import { admin } from "stores/portal"
import { get } from "svelte/store"

async function isCloud() {
   let adminStore = get(admin)
   if (!adminStore.loaded) {
       await admin.init()
      adminStore = get(admin)
   }
  
   return adminStore.cloud

}

```